internal/legacy/dockerregistry/inventory_test.go

```diff
@@ -835,107 +835,6 @@ func TestSplitByKnownRegistries(t *testing.T) {
 	}
 }
 
-func TestCommandGeneration(t *testing.T) {
```

Comment:
shouldn't we completely remove the invocations to the functions this test is checking before removing the test completely?

Reply:
I think its only GetDeleteCmd

Reply:
Done 👍

Reply:
Thanks!